paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -206,7 +206,8 @@ void CpuPassStrategy::EnableMKLDNN() {
              "reshape_transpose_matmul_mkldnn_fuse_pass",  //
              "matmul_transpose_reshape_fuse_pass",         //
              // Disabled due to topology-dependent speed-up
-             // "fc_mkldnn_pass",
+             //"fc_mkldnn_pass",
+             //"fc_act_fuse_pass",
```

Comment:
Should the pass be enabled conditionally somewhere?

Reply:
@jczaja Is this pass for native fc or fc_mkldnn? How about the name `fc_act_mkldnn_fuse_pass`?

Reply:
@luotao1 Name of pass was modified as suggested.